.yarn/patches/postgres-shift-npm-0.1.0-9342b5f6f6.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/index.js b/index.js
+index 0115c8519e8c756ee24f50418eb2d85e8fc5c617..cf5cac0df35edf705c51ef7ed0ed489b69c0777e 100644
+--- a/index.js
++++ b/index.js
+@@ -1,9 +1,9 @@
+-import fs from 'fs'
+-import path from 'path'
++const fs = require('fs')
++const path = require('path')
+ 
+ const join = path.join
+ 
+-export default async function({
++module.exports = async function({
+   sql,
+   path = join(process.cwd(), 'migrations'),
+   before = null,
+@@ -48,7 +48,7 @@ export default async function({
+   }) {
+     fs.existsSync(join(path, 'index.sql')) && !fs.existsSync(join(path, 'index.js'))
+       ? await sql.file(join(path, 'index.sql'))
+-      : await import(join(path, 'index.js')).then(x => x.default(sql)) // eslint-disable-line
++      : await require(join(path, 'index.js'))(sql) // eslint-disable-line
```

Comment:
We don't need to `await` here anymore as `require` is syncronous.

```suggestion
+      : require(join(path, 'index.js'))(sql) // eslint-disable-line
```

Reply:
Interesting. I've actually used a previous non-ESM version of the library for this change https://github.com/porsager/postgres-shift/blob/c841edd6f1002db6713ce961e8339b298e8946e8/index.js#L51

Reply:
I see. Let's keep it as is then.